src/dashboard/tests/test_api.py

```diff
@@ -576,3 +579,150 @@ def test_copy_metadata_files_api(mocker):
         "message": "Metadata files added successfully.",
         "error": None,
     }
+
+
+@pytest.fixture
+def username():
+    return "test"
+
+
+@pytest.fixture
+def password():
+    return "test"
+
+
+def dashboard_login_and_setup(client, django_user_model, username, password):
+    django_user_model.objects.create_user(username=username, password=password)
+    client.login(username=username, password=password)
+    helpers.set_setting("dashboard_uuid", "test-uuid")
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "transfer_type,expected_directory,alt_dir",
```

Comment:
What's the purpose of `alt_dir`?

Reply:
Ah, not too much, I'll remove! :slightly_smiling_face: